### ISSUE TITLE
Browsing HDF5 files interactively

### ISSUE THREAD

**Issue by jan-janssen** [h5io/h5io#67]
To extend the functionality of the `list_file_contents()` function, I developed an interactive file browser for HDF5 files created using h5io. The new module is available at https://github.com/pyiron/h5io_browser

I was wondering, if this repository should be transferred to the h5io organisation, or if it should remain external.

**Comment by larsoner** [comment]
I'm fine either way. If you want to transfer the repo I can make you an admin of the h5io org

**Comment by jan-janssen** [comment]
> I'm fine either way. If you want to transfer the repo I can make you an admin of the h5io org

On the one hand I have the hope that by publishing it as part of the h5io organisation, more developers who are currently using h5io might also try h5io_browser and give me feedback on how to further improve it. On the other hand the Github actions and the `setup.py` / `pyproject.toml` is currently rather different from the one used for h5io. Inside a given Github organisation it is most likely beneficial to have a similar setup for all repositories in a given organisation. So I am open to adjust both to match the h5io repository. 

In summary, I am happy to transfer the h5io_browser repository to the h5io organisation.

**Comment by larsoner** [comment]
I am not sure who all uses h5io other than MNE-Python (what I originally worked on `h5io` for). But sure keeping them together would help. Feel free to modernize both while you're at it if you want -- in MNE-Python nowadays we use `hatchling` and `hatch-vcs` and if we adopted that here we wouldn't need to manually change `_version.py` anymore, which would be nice.

I just invited you to be an owner of the org
